apple-nav/src/components/tabs.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Tab from "./tab"
+
+
+const Tabs=(props)=> {
+    return (
+      <div style={{display:"flex",justifyContent:"space-around",marginTop:"20px"}}>
+       {props.tabData.map((tab,i)=>(
+           <Tab key={i} tab={tab}/>
+       ))}
```

Comment:
You could also create separate CSS files or utilize styled components instead of placing styles inline directly into the `style` attribute.